analyses/molecular-subtyping-EPN/02_ependymoma_generate_all_data.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+# coding: utf-8
+# Author Teja Koganti (D3B)
+
+
+import pandas as pd
+import numpy as np 
+import zipfile
+import  pyreadr
+import statistics
+import rpy2.robjects as robjects
+from rpy2.robjects import pandas2ri
+pandas2ri.activate()
+
+# File to write all the data in analyses/molecul-EPN/results folder 
+outfile = open("analyses/molecular-subtyping-EPN/results/EPN_all_data.tsv", "w")
+
+# Reading GISTIC broad_values file for CNA 
+#CNA = pd.read_csv("/Users/kogantit/Documents/OpenPBTA_tickets/ependymoma_subtyping_ticket/2019-12-10-gistic-results-cnvkit/broad_values_by_arm.txt", sep="\t")
+#CNA = CNA.set_index("Chromosome Arm")
+zip=zipfile.ZipFile('data/pbta-cnv-cnvkit-gistic.zip')
+CNA=pd.read_csv(zip.open('2019-12-10-gistic-results-cnvkit/broad_values_by_arm.txt'), sep="\t")
+CNA = CNA.set_index("Chromosome Arm")
+
+count=0
```

Comment:
Is this `count` used anywhere?
I see you adding to it later, but nowhere that the final value is used.

Reply:
Removed this now